tests/regression/CMakeLists.txt

```diff
@@ -84,6 +85,8 @@ add_test_pocl(NAME "regression/test_issue_1435" COMMAND "test_issue_1435")
 
 add_test_pocl(NAME "regression/test_issue_1525" COMMAND "test_issue_1525" LLVM_FILECHECK "test_issue_1525.fc" WORKITEM_HANDLER "loopvec")
 
+add_test_pocl(NAME "regression/test_issue_1608" COMMAND "test_issue_1608")
```

Comment:
Does this test now pass with all supported LLVM versions with the CPU device ?

Because the CI machines for CPU-device only run tests labeled with "cpu" label (which is a subset of all tests). This test does not have any label, hence it won't be picked up by any CI as it is now.

Reply:
label can be added later.